content/nginx-one/how-to/config-sync-groups/add-file-csg.md

```diff
@@ -0,0 +1,70 @@
+---
+docs: 
+doctypes:
+    - task
+tags:
+    - docs
+title: Add a file to a Config Sync Group
+toc: true
+weight: 400
+---
+
+
+## Overview
+
+{{< include "nginx-one/add-file/overview.md" >}}
+
```

Comment:
```suggestion

---

```

Reply:
@ADubhlaoich This should be lintable in some way... If you don't know how, I'll open an issue.

Reply:
It's definitely lintable, we just don't have any tooling in place for it. It's something I can add with my upcoming work.